deployment-react/pom.xml

```
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.mcollovati</groupId>
+        <artifactId>quarkus-hilla-parent</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>quarkus-hilla-react-deployment</artifactId>
```

```suggestion
    <artifactId>quarkus-hilla-deployment-react</artifactId>
```
Following the same naming scheme